### DISCUSSION TITLE
winmail.dat

### DISCUSSION THREAD

**Discussion by chris-yooo** [the-djmaze/snappymail#1012] [Ideas]
the abbility to open winmail.dat files :)

**Comment by the-djmaze** [comment]
TNEF is something i've looked into.
But i don't see worth the time and effort as it should not be used anymore and i also never see such messages these days.

**Reply by chris-yooo** [reply]
my it companion curses every day :D - they get minimum 3 mails at day that are send with old outlook's.

and use in the moment a **itty website to convert them :S


we would appreciate it

**Comment by noseshimself** [comment]
I'm still getting lots of that sh*censored*. Basically whenever anybody tries sending RTF mails (guess what happens when you are using Word as mail client...) via Exchange you will get a TNEF.

I can't really see adding https://github.com/koodaamo/tnefparse or libytnef as wasted effort. We will have to bear it for another 20 years just like I had to read DOS 3.x floppies a few weeks ago.

**Reply by the-djmaze** [reply]
Ok, i will see what i can do as the RTF also needs to be converted.

In the meanwhile you can listen to some floppies 
https://m.youtube.com/watch?v=cM_sAxrAu7Q

**Comment by the-djmaze** [comment]
I was just reading some winmail.dat files, i've found online.

Very funny that they contain personal info.
For example i found in there: `C:\Documents and Settings\{$USERNAME}\Local Settings\Application Data\Microsoft\Outlook\Outlook.pst`

I will keep on digging how to extract these files in PHP without overhead.

**Comment by the-djmaze** [comment]
Here's something to read the RTF from the winmail.dat
No idea if it will work, i need emails as example

https://github.com/the-djmaze/snappymail/blob/565e8f0fb7f72698c445f87056f0723afe49be18/snappymail/v/0.0.0/app/libraries/RainLoop/Actions/Messages.php#L518-L540

**Comment by chris-yooo** [comment]
please send me an mail [chris](https://github.com/chris-yooo)

greetz
